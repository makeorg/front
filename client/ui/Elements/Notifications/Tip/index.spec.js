import React from 'react';
import renderer from 'react-test-renderer';
import { Tip } from './index';

jest.mock('./style', () => ({
  TipWrapperStyle: 'TipWrapperStyle',
  TipCrossStyle: 'TipCrossStyle',
  TriangleStyle: 'TriangleStyle',
  LinkStyle: 'LinkStyle',
  TipContainerStyle: 'TipContainerStyle',
}));

jest.mock('Client/ui/Svg/elements', () => ({
  SvgClose: 'SvgClose',
}));

jest.mock('../AccessibilityElements', () => ({
  ScreenReaderItemStyle: 'ScreenReaderItemStyle',
}));

describe('TagTooltip', () => {
  it('must match the snapshot with default Props', () => {
    const component = renderer.create(<Tip />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
