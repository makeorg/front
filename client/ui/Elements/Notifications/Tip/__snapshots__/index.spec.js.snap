// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagTooltip must match the snapshot with default Props 1`] = `
Array [
  <TriangleUpStyle />,
  <TipWrapperStyle
    isFirstSequenceVote={false}
  >
    <TipCrossStyle
      aria-label="common.notifications.icons.close"
      onClick={[Function]}
    >
      <SvgClose
        aria-hidden={true}
        focusable="false"
      />
    </TipCrossStyle>
    <NotificationIcon
      context="tip"
      level="information"
    />
    common.notifications.vote_on_proposal
  </TipWrapperStyle>,
]
`;
